pkg/controller/wip/controller.go

```diff
@@ -382,11 +383,26 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 	glog.V(4).Infof("Creating client for Broker %v, URL: %v", broker.Name, broker.Spec.URL)
 	brokerClient := c.brokerClientCreateFunc(broker.Name, broker.Spec.URL, username, password)
 
-	// TODO: parameters
+	parameters := make(map[string]interface{})
+	if len(instance.Spec.Parameters.Raw) > 0 {
+		err = yaml.Unmarshal([]byte(instance.Spec.Parameters.Raw), &parameters)
+		if err != nil {
+			glog.Errorf("Failed to unmarshal Instance parameters\n%s\n %v", instance.Spec.Parameters, err)
+			c.updateInstanceCondition(
+				instance,
+				v1alpha1.InstanceConditionReady,
+				v1alpha1.ConditionFalse,
+				"ErrorWithParameters",
```

Comment:
Yeah, I can do that, just looking at the file it seemed to not use consts so was trying to be consistent with the file convention.